Classes/Console/Database/Process/MysqlCommand.php

```diff
@@ -51,7 +51,7 @@ public function __construct(array $dbConfig, ConsoleOutput $output = null)
     public function mysql(array $additionalArguments = [], $inputStream = STDIN, $outputCallback = null, $interactive = false)
```

Comment:
@mbrodala First of all, thanks for your optimisations! :)

Why do you think we need the extra function?

This is how I understand it:
All connections that have `mysql` in the driver are read from `$GLOBALS[‘TYPO3_CONF_VARS’][‘DB’][‘Connections’]`. 

See https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/Configuration/Typo3ConfVars/DB.html#confval-typo3-conf-vars-db-connection-name-driver

Reply:
The extra function is necessary to call the `mariadb` CLI client instead of the [compat-layer `mysql`](https://mariadb.com/kb/en/mysql-command-line-client/). In the long term MariaDB we can assume that MariaDB will drop the latter the same way as it did for `mysqldump`.